CHANGES.rst

```diff
@@ -228,6 +228,14 @@ API Changes
   - Moved ``units.cgs.emu`` to ``units.deprecated.emu`` due to ambiguous
     definition of "emu". [#4918, #5906]
 
+  - ``jupiterMass``, ``earthMass``, ``jupiterRad``, and ``earthRad`` no longer
+    have their prefixed units included in the standard units.  If needed, they
+    can still  be found in ``units.deprecated``. [#5661]
+    
+  - ``solLum``,``solMass``, and ``solRad`` no longer have  their prefixed units
+    included in the standard units.  If needed, they can still be found in  
+    ``units.vounit_needed``, and are enabled by default. [#5661]
```

Comment:
I'll try going with `required_by_vounit` .  I favor that over `vounit_with_prefix` only because it's *possible* that in the future we might put non-prefix units there (if there's ever a VOUnit 2.0 or another change in what people think should be in `astropy.units`).